nucypher/cli/painting.py

```diff
@@ -0,0 +1,184 @@
+"""
+This file is part of nucypher.
+
+nucypher is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+nucypher is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+
+import click
+import maya
+
+import nucypher
+from constant_sorrow.constants import NO_KNOWN_NODES
+from nucypher.config.characters import UrsulaConfiguration
+from nucypher.config.constants import SEEDNODES
+
+#
+# Art
+#
+
+BANNER = """
+                                  _               
+                                 | |              
+     _ __  _   _  ___ _   _ _ __ | |__   ___ _ __ 
+    | '_ \| | | |/ __| | | | '_ \| '_ \ / _ \ '__|
+    | | | | |_| | (__| |_| | |_) | | | |  __/ |   
+    |_| |_|\__,_|\___|\__, | .__/|_| |_|\___|_|   
+                       __/ | |                    
+                      |___/|_|  
+
+    version {}
+
+""".format(nucypher.__version__)
+
+
+#
+# Paint
+#
+
+def build_fleet_state_status(ursula) -> str:
+    # Build FleetState status line
+    if ursula.known_nodes.checksum is not NO_KNOWN_NODES:
+        fleet_state_checksum = ursula.known_nodes.checksum[:7]
+        fleet_state_nickname = ursula.known_nodes.nickname
+        fleet_state_icon = ursula.known_nodes.icon
+        fleet_state = '{checksum} ⇀{nickname}↽ {icon}'.format(icon=fleet_state_icon,
+                                                              nickname=fleet_state_nickname,
+                                                              checksum=fleet_state_checksum)
+    elif ursula.known_nodes.checksum is not NO_KNOWN_NODES:
```

Comment:
Did you mean
```suggestion
    elif ursula.known_nodes.checksum is NO_KNOWN_NODES:
```

`is not` is already checked in the if above.

Reply:
Correct - good eye.

We'll start to need some end-to-end tests (ideally, even browser-automation if indeed we're going to do web admin type stuff).